packages/components/src/components/Note/Note.module.css

```diff
@@ -0,0 +1,71 @@
+@import "@/styles";
+
+.root {
+  border-radius: var(--note--border-radius);
+  border: var(--note--border-width--default) var(--note--border-style)
```

Comment:
Den `border-style` auch mit in die Design-Tokens zu übernehmen finde ich (vielleicht!) nicht passend, da die Werte (`solid`, usw.) ja reine CSS Werte sind. Ähnlich wie bei

```
display: grid;
```

und nicht

```
display: var(---note-display);
```

Ist aber grenzwertig Deine Meinung?

Reply:
ich häts als Variable gemacht, weil wir ja ganz selten auch einen anderen border style verwenden. aber momentan wirklich nur in einem Fall, nämlich beim aktuellen Tarif border:dashed, und das wär ja dann ein Wert den UX uns vorgeben würde oder?

Reply:
Mmm... ok. Kannst du es dann auf drei Props verteilen? Ist schneller zu durchschauen und wartungsfreundlicher.

```css
border-width: var(--note--border-width--default);
border-style: ...
```